roles/openshift_manage_node/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+- name: Handle unscheduleable node
+  delegate_to: "{{ openshift_first_master }}"
```

Comment:
@brenton I suspect so, it probably creeped in there while it was refactored back and forth as well.